openedx/core/djangoapps/content_tagging/utils.py

```diff
@@ -93,7 +93,7 @@ def get_library_orgs(self, user, org_names: list[str]) -> list[Organization]:
         if library_orgs is None:
             library_orgs = {
                 library.org.short_name: library.org
-                for library in get_libraries_for_user(user).select_related('org').only('org')
+                for library in get_libraries_for_user(user).select_related('org').only('org', 'learning_package')
```

Comment:
Nit: I would appreciate a comment here as a reminder, like:

`# We don't actually need learning_package here but it's already included by get_libraries_for_user() so we need to include it in .only() or we get an ORM error.`

Reply:
Yeah good call 👍🏻